### ISSUE TITLE
Documentation: How to work around missing CDK features (overrides)

### ISSUE THREAD

**Issue by ccfife** [awslabs/aws-cdk#2294]
This is a new topic for the how to section, and is largely covered "Accessing the AWS CloudFormation Layer" topic in the current guide. We just need to review the topic and make sure it is at the quality bar we want. We've learned that customers need to use the "escape hatch" to address specific use cases the L2 don't address yet.

**Comment by fulghum** [comment]
Blocked by #2100

**Comment by fulghum** [comment]
We should rescope this issue to take out the incorrect information and create a new issue to add the fixed case after 2100 completes.

**Comment by Doug-AWS** [comment]
If I hear Rico correctly, we still support overriding untyped properties, but 2100 is for the typed. I'll have a review available shortly, after I send out the sections for Mitch to review.

**Comment by Doug-AWS** [comment]
I've commented-out the incorrect info and should have the docs updated by tomorrow.

**Comment by Doug-AWS** [comment]
We're talking about this section: https://docs.aws.amazon.com/cdk/latest/guide/cfn_layer.html

**Comment by Doug-AWS** [comment]
Moving all guide issues to doc repo.
